mobileapps/tests.py

```diff
@@ -1612,3 +1612,38 @@ def test_mobileapps_organization_theme_with_no_default_images(self):
 
         for key, value in settings.ORGANIZATION_HEADER_BG_IMAGE_SIZES_MAP.items():
             self.assertNotIn('image_url_{}'.format(key), response.data['header_bg_image'])
+
+    def test_mobileapps_organization_theme_remove_logo_image(self):
+        organization_theme = Theme.objects.create(
+            name='Blue',
+            logo_image_uploaded_at=TEST_LOGO_IMAGE_UPLOAD_DT,
+            header_bg_image_uploaded_at=TEST_HEADER_BG_IMAGE_UPLOAD_DT,
+            active=True,
+            organization=self.organization1,
+        )
+
+        response = self.do_delete(reverse(
+            'mobileapps-organization-themes-remove-image', kwargs={
+                'theme_id': organization_theme.id,
+                'attribute': 'logo_image'
+            }
+        ))
+        self.assertEqual(response.status_code, 200)
+
+        response = self.do_delete(reverse(
+            'mobileapps-organization-themes-remove-image', kwargs={
+                'theme_id': organization_theme.id,
+                'attribute': 'header_bg_image'
+            }
+        ))
+        self.assertEqual(response.status_code, 200)
+
+        response = self.do_get(reverse(
```

Comment:
Can we add a test which hits image deletion url with invalid `theme_id` and valid attribute and then valid `theme_id` and invalid attribute?